examples/gpt/deepspeed_hf.py

```diff
@@ -171,6 +173,8 @@ def loss_fn(outputs, labels):
 
     loader = RepeatingLoader(train_loader)
 
+    model.load_checkpoint("checkpoint", "global_step40")
```

Comment:
I suppose this is for testing checkpoint? If so, I'd suggest removing it and creating a unit test.

Reply:
oh this is added accidentally. This shouldn't be included in slapo repo.